mpt/impl/imap-mailbox/core/src/main/resources/org/apache/james/imap/scripts/UidSearchCombinations.test

```diff
@@ -632,7 +632,7 @@ C: Rhubarb!
 S: \* 9 EXISTS
 S: \* 9 RECENT
 S: A72 OK (\[.+\] )?APPEND completed\.
-C: A73 APPEND testmailbox {11976+}
+C: A73 APPEND testmailbox {11975+}
```

Comment:
Could you try if these modifications are still needed if you use not an External James ?